pkgs/top-level/all-packages.nix

```diff
@@ -19485,8 +19485,9 @@ in
 
   termtosvg = callPackage ../tools/misc/termtosvg { };
 
-  tesseract = callPackage ../applications/graphics/tesseract { };
-  tesseract_4 = lowPrio (callPackage ../applications/graphics/tesseract/4.x.nix { });
+  inherit (import ../applications/graphics/tesseract pkgs)
```

Comment:
I would prefer to pass `callPackage` from the top scope, not `pkgs`.

Reply:
Like so? https://github.com/erikarvstedt/nixpkgs/commit/d7f298d4526bafd50149be668ffd89b0e21e32c7

Reply:
Yes